externs/ngeox.js

```diff
@@ -89,28 +97,83 @@ ngeox.profile.ProfileOptions.prototype.hoverCallback;
 
 
 /**
+ * @type {function(Object, Object)|undefined}
+ */
+ngeox.profile.ProfileOptions.prototype.poisProvider;
+
+
+/**
+ * @type {function(Object)|undefined}
+ */
+ngeox.profile.ProfileOptions.prototype.metasProvider;
+
+
+/**
  * @type {function()|undefined}
  */
 ngeox.profile.ProfileOptions.prototype.outCallback;
 
 
 /**
- * Profile data extractor.
+ * Elevation data extractor.
  * @typedef {{
  *   z: function(Object): number,
  *   dist: function(Object): number
  * }}
  */
-ngeox.profile.ProfileExtractor;
+ngeox.profile.ElevationExtractor;
+
+
+/**
+ * @type {function(Object): number}
+ */
+ngeox.profile.ElevationExtractor.prototype.z;
+
+
+/**
+ * @type {function(Object): number}
+ */
+ngeox.profile.ElevationExtractor.prototype.dist;
+
+
+/**
+ * Profile data extractor.
+ * @typedef {{
+ *   id: function(Object): string,
+ *   dist: function(Object): number,
+ *   alt: function(Object, number=): number,
```

Comment:
alt is altitude? You use elevation elsewhere. Don't you want to use the same word everywhere?